.github/workflows/build.yml

```diff
@@ -15,10 +15,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        rails: [7.1.1, 7.0.8, 6.1.7]
+        rails: [7.2.0, 7.1.1, 7.0.8, 6.1.7]
         ruby: [3.3, 3.2, 3.1, 3.0.6, jruby-9.4]
```

Comment:
I think removing 3.0.6 from this line will suffice to move us forward... 
not sure if we should add an explicit include for 3.0.x below or just ignore?

Reply:
I added an explicit exclude in the end.